superset/templates/superset/basic.html

```diff
@@ -60,7 +60,7 @@
       {% endif %}
 
     {% endblock %}
-
+    <script>window.__SUPERSET_DEPLOYMENT_PREFIX__ = "{{ assets_prefix }}";</script>
```

Comment:
As I understand `assets_prefix `is initialized in "/superset/extensions/__init__.py" using the `STATIC_ASSETS_PREFIX `configuration variable, but i cannot see how and where `ASSET_BASE_URL` (Dockerfile) is assigned to be used after in the `withPrefix()` function